core/src/main/java/com/oath/oak/EntrySet.java

```diff
@@ -281,9 +287,8 @@ private void setEntryFieldLong(int item, OFFSET offset, long value) {
      * CAS from 'expectedIntValue' to 'newIntValue' for field at specified offset
      * of given int-field in the entries array
      */
-    private boolean casEntriesArrayInt(
-        int intFieldIdx, OFFSET offset, int expectedIntValue, int newIntValue) {
-        return unsafe.compareAndSwapInt(entries,
+    private boolean casEntriesArrayInt(int intFieldIdx, OFFSET offset, int expectedIntValue, int newIntValue) {
```

Comment:
Are there no boundaries on long code line?

Reply:
Sure there is. See the `codestyle/checkstyle.xml`. I set it to 120. It seems the correct length because most of the code conforms to this length, and it is also Intellij defaults.
This line (and now all the other lines) conforms to this limitation.